static-metric/Cargo.toml

```diff
@@ -17,9 +17,9 @@ include = [
 proc-macro = true
 
 [dependencies]
-syn = { version = "0.13", features = ["full", "extra-traits"] }
-proc-macro2 = "0.3"
-quote = "0.5"
+syn = { version = "0.15", features = ["full", "extra-traits"] }
```

Comment:
syn has version 1.0.3: https://docs.rs/syn/1.0.3/syn/

Reply:
IMO we should update TiKV's `syn` to 1.0 too.

Reply:
The issue is not what version TiKV is using, but what version it's deps are using (Serde, Prost, slog, structopt, and more). We *should* send PRs to update them all, but I don't want to take that on right now when there is low-hanging fruit in just updating Prometheus to use a more recent version. Let's not let perfect be the enemy of good here.

Reply:
ping @breeswish

Reply:
Ok

Reply:
But what's the purpose of updating these versions?